index.js

```diff
@@ -162,6 +163,19 @@ Consumer.prototype._processMessage = function (message, cb) {
       } else {
         consumer.emit('processing_error', err, message);
       }
+
+      if (consumer.terminateVisibilityTimeout) {
+        consumer.sqs.changeMessageVisibility({
+          QueueUrl: consumer.queueUrl,
+          ReceiptHandle: message.ReceiptHandle,
+          VisibilityTimeout: 0
+        }, function(err) {
+          if (err)
```

Comment:
Just a style comment, but please put consumer on the same line as the if or add braces.